src/styles/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$nav-color: rgba(0, 128, 128, .5);
```

Comment:
Thank you for creating a variables scss file and as well implementing them into css file.